src/main/resources/subscriptionsTemplate.json

```diff
@@ -19,7 +20,10 @@
                 }
             ]
         }
-    ]
+
+    ],
+    "subscriptionName" : "Subscription_Test",
```

Comment:
Remove subscriptionName line here. You have already one subscriptionName line in the beginning:  "subscriptionName" : "Subscription1", ....